### ISSUE TITLE
Request - Name, icon and Y-axis description optional?

### ISSUE THREAD

**Issue by janser01** [aex351/home-assistant-neerslag-card#7]
Thank you for this awesome card!

Would it be possible to make the name, icon and Y-axis description optional?

I'm using the card in a vertical stack configuration and it looks ok but I think when you can suppress the icon, the name and the Y-axis description it looks even better!

<img width="401" alt="Screenshot 2021-03-17 at 17 10 02" src="https://user-images.githubusercontent.com/54860469/111500282-1d0f4d00-8744-11eb-9c30-fe143c8aec85.png">

**Comment by coltju** [comment]
Agree, this would be very cool to control. Maybe add some possibilty's to change the Y axis labels. it gets quite crowded, so make the interval lower. like 4 labels.
